include/mapnik/util/math.hpp

```diff
@@ -25,8 +25,12 @@
 
 #include <mapnik/config.hpp>
 
+#include <cmath>
+
 namespace mapnik { namespace util {
 
+constexpr double pi = 3.14159265358979323846;
+
```

Comment:
`M_PI` is not part of c++ so yes, defining it in `include/mapnik/util/math.hpp` and removing MSVC related #if/def in `global.hpp` makes sense to me. One thing to be aware third party libs are using  `M_PI` from math.h, agg has its own #define ... We just have to be consistent